pkg/kancontroller/kancontroller.go

```diff
@@ -37,8 +38,30 @@ import (
 	"github.com/kanisterio/kanister/pkg/kube"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/resource"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	kanisterMetricsEnv = "KANISTER_METRICS_ENABLED"
+)
+
+// metricsEnabled checks if the feature flag for kanister metrics is enabled
+// If the environment variable is not set, then it returns a default
+// "false" value.
+func metricsEnabled() bool {
+	metricsEnabled, ok := os.LookupEnv(kanisterMetricsEnv)
+	if !ok {
+		log.Error().Print("KANISTER_METRICS_ENABLED env variable not set")
```

Comment:
```suggestion
		log.Error().Print("KANISTER_METRICS_ENABLED env variable not set")
```

It's not actually an error right?

Reply:
Yes, it's not an error. I've changed it to use `log.Print()` instead of `log.Error().Print()`

Reply:
Since the helm flag has a value by default (false), there is never a scenario where `KANISTER_METRICS_ENABLED` env variable is not set, unless an admin decides to forcefully delete the environment variable. From that perspective it could be an error. 
Please let me know if this reasoning makes sense - if not, we could go ahead with just using `log.Print()` - in the end which should log this scenario for any debugging purpose.